submarine-server/server-core/src/main/java/org/apache/submarine/server/utils/GitRequestUtil.java

```diff
@@ -32,8 +32,8 @@
 import java.util.Date;
 import java.util.Map;
 
-public class HttpRequestUtil {
-  private static final Logger LOG = LoggerFactory.getLogger(HttpRequestUtil.class);
+public class GitRequestUtil {
```

Comment:
Do we need to change HttpRequestUtil to GitRequestUtil?

Reply:
HttpRequestUtil is a special class for git http request, Change to GitHttpRequest.